modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2304,11 +2304,14 @@ export class Wallet implements IWallet {
       buildParams.recipients = params.enableTokens.map((token) => {
         return {
           tokenName: token.name,
-          address: token.address,
+          address: token.address || this._wallet.coinSpecific!.baseAddress!,
+          amount: '0',
```

Comment:
do we need to set this explicitly?
feel like for coins that want to leverage this function, they'll need to update WP accordingly to handle missing `address` and `amount` when building token enablement transactions

Reply:
if we need this, lets do the following:

ensure user is not passing `amount`. if they are, throw an error like `Can not specify amount when enabling tokens`

additionally, lets nil-check `this._wallet.coinSpecific!.baseAddress` and throw error accordingly:
```
if (!this._wallet.coinSpecific?.baseAddress) {
   throw new Error('Wallet does not have base address, must specify with token param') ;
}
```

Reply:
The user cannot specify amount since we forbid them from specifying `recipients`. Only `address` and `name` are in `TokenEnablement`